packages/wagmi/package.json

```diff
@@ -43,6 +43,7 @@
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
   },
   "dependencies": {
+    "@web3modal/siwe": "3.2.0",
```

Comment:
`@web3modal/siwe` is a peer / optional dependency, so we shouldn't include it here. For dev purposes (i.e. types) we can add it to devDependencies